integration/localnet/Makefile

```diff
@@ -76,7 +76,7 @@ stop:
 
 .PHONY: load
 load:
-	go run --tags relic ../benchmark/cmd/manual -log-level info -tps 1,0,10,0,100 -tps-durations 30s,10s,30s,10s
+	go run --tags relic ../benchmark/cmd/manual -log-level info -tps 1,10,100 -tps-durations 30s,30s
```

Comment:
0's are not needed.  This was previously used because we did not wait for benchmark to stop and these added 10s sleeps between runs.